pkg/testing/integration/program_test.go

```diff
@@ -91,3 +92,66 @@ func TestDepRootCalc(t *testing.T) {
 	dep = getRewritePath("github.com/pulumi/pulumi-auth0/sdk", "gopath", "/my-go-src")
 	assert.Equal(t, "/my-go-src/pulumi-auth0/sdk", filepath.ToSlash(dep))
 }
+
+func TestGoModEdits(t *testing.T) {
+	t.Parallel()
+
+	depRoot := os.Getenv("PULUMI_GO_DEP_ROOT")
+	gopath, err := GoPath()
+	contract.AssertNoError(err)
```

Comment:
Just curious: Why use `contract.AssertNoError(err)` here vs. `require.NoError(t, err)` from `"github.com/stretchr/testify/require"`?

Reply:
No good reason, fixed up the commit with `require`.